test/robot_test.js

```diff
@@ -28,8 +28,7 @@ describe('Robot', function () {
       warnOnUnregistered: false
     })
     mockery.registerMock('hubot-mock-adapter', require('./fixtures/mock-adapter'))
-    this.robot = new Robot(null, 'mock-adapter', true, 'TestHubot')
-    this.robot.alias = 'Hubot'
+    this.robot = new Robot({adapter: 'mock-adapter', httpd: false, name: 'TestHubot', alias: 'Hubot'})
```

Comment:
Even though it’s deprecated, I would suggest we keep a test for the old usage to make sure we don’t accidentally break it in future feature/fix releases. It will also increase the test coverage and make coveralls status pass :)